smithy-openapi/src/main/java/software/amazon/smithy/openapi/OpenApiConfig.java

```diff
@@ -330,6 +329,14 @@ public void setUseIntegerType(boolean useIntegerType) {
         this.useIntegerType = useIntegerType;
     }
 
+    public String getVersion() {
+        return this.version;
+    }
+
+    public void setVersion(String version) {
+        this.version = version;
```

Comment:
Let's do the version validation here instead, including an `Objects.requireNonNull`

Reply:
resolved.